peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -145,6 +145,25 @@ <h1 class="py-4">{{ document.title }}</h1>
                           </dd>
                         {% endif %}
                       {% endblock %}
+
+                      {% block document-metadata-media-summary-download %}
+                        {% if document.attachedfiles_set %}
+                          <dt>{% translate 'Attachments' %}</dt>
```

Comment:
I'd like this done a bit differently, please. The user shouldn't have to understand our concept of attachments. Rather, show each attached file as its own metadata entry.

```
Language          English
Media summary     Download PDF (3.2MB)
Media summary     Download DOCX (1MB)
Original document Download RTF (1.2MB)
  
```